pkg/lib/machine-info.js

```diff
@@ -25,17 +25,17 @@ export const cpu_ram_info = address =>
             .then(text => {
                 const info = { };
                 const memtotal_match = text.match(/MemTotal:[^0-9]*([0-9]+) [kK]B/);
-                const total_kb = memtotal_match && parseInt(memtotal_match[1], 10);
+                const total_kb = parseInt(memtotal_match?.[1], 10);
```

Comment:
This looks unsafe. `?.` just results in `undefined`, and we don't want to call `parseInt(undefined)`. Please keep the old code.

Reply:
Done